tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr

```diff
@@ -13,7 +13,8 @@ fn main(
   // Receiver public key
   to_pubkey_x: Field, 
   to_pubkey_y: Field, 
-) -> pub [Field; 2] {
+) -> pub [Field; 2]
+{
```

Comment:
Lets add this as an issue and exclude this file from the PR

Reply:
#3310